contentcuration/contentcuration/views.py

```diff
@@ -46,22 +46,57 @@ def unsupported_browser(request):
 def unauthorized(request):
     return render(request, 'unauthorized.html')
 
+def channel_page(request, channel, allow_edit=False):
```

Comment:
Moved to separate method to allow code reuse for entering view-only and edit pages